libr/anal/Makefile

```diff
@@ -4,7 +4,7 @@ EXTRA_TARGETS+=do
 EXTRA_CLEAN=doclean
 
 NAME=r_anal
-DEPS=r_util r_reg r_syscall
+DEPS=r_util r_reg r_syscall r_asm
```

Comment:
if r_asm is compiled without xtensa support, these *.o files won't be compiled either, right?